Dockerfile

```diff
@@ -615,7 +615,7 @@ RUN \
   --enable-libdavs2 \
   --enable-libxavs2 \
   --enable-libmodplug \
-  --enable-libuavs3d \
+#  --enable-libuavs3d \
```

Comment:
Hopefully can be enabled again after https://github.com/wader/static-ffmpeg/pull/153